apps/meteor/app/api/server/lib/rooms.ts

```diff
@@ -39,7 +39,13 @@ export async function findAdminRooms({
 
 	const { cursor, totalCount } = result;
 
-	const [rooms, total] = await Promise.all([cursor.sort(sort || { default: -1, name: 1 }).toArray(), totalCount]);
+	const [rooms, total] = await Promise.all([
+		cursor
+			.collation({ locale: 'en' })
```

Comment:
```suggestion
			.collation({ caseFirst: 'off' })
```

Reply:
Actually, locale is a required field in collation :grimacing: so dismiss this suggestion, let's move on with the other approach